tests/modules/services/window-managers/sway/sway-null-package.conf

```diff
@@ -1,4 +1,4 @@
-font pango:monospace 8
+font pango:monospace 8.000000
```

Comment:
this isnt right, it should remain `8`

Reply:
see discussion here: https://github.com/nix-community/home-manager/pull/1950#discussion_r619927313

Reply:
ha sry had not seen.

Reply:
no worries :) thanks for the review!